docs/user-guide/aroma/sd-preparation.md

```diff
@@ -22,3 +22,61 @@ We will now place the required Aroma files on the SD Card.
     - The two `wiiu` folders should be merged if not done automatically.
 
 ?> If your computer asks you to overwrite existing files on your SD Card, you will need to click yes.
+----------
+
+### SD Card Layout {docsify-ignore}
+
+<details>
+<summary>Click here to show the final SD Card layout.</summary>
+
+```
+💾sd:
+ ┗ 📂wiiu
+   ┣ 📂apps
+   ┃ ┣ 📂homebrew_launcher
```

Comment:
I don't think we need to list the `homebrew_launcher` in the aroma structure, this might only lead to confusion.